packages/core/app-layout/src/components/sidebar/AppSidebar.vue

```diff
@@ -454,23 +454,23 @@ onBeforeUnmount(() => {
 
 .sidebar-top {
   align-items: center;
-  color: #fff;
+  color: $kui-color-background;
```

Comment:
Wrong token, `color-background` token for color (were you looking for `kui-color-text-inverse`?)

Reply:
fixed